### ISSUE TITLE
Making sure that master is not schedulable

### ISSUE THREAD

**Issue by florianmoss** [ryanhay/ocp4-metal-install#3]
Hi Ryan

I would suggest making it clear that the:

`sed -i 's/mastersSchedulable: true/mastersSchedulable: false/' ~/ocp-install/manifests/cluster-scheduler-02-config.yml`

command is not optional. I missed it the first time and the following instructions only work when the commands were run before. Great guide though, highly appreciated!

**Comment by ryanhay** [comment]
Hi Florianmoss, thanks for the feedback. What didn't work for you when the masters weren't schedulable? Did you bring the worker nodes up and add them to the cluster during installation so the pods that couldn't be scheduled on the master could be scheduled?

**Comment by florianmoss** [comment]
Hi Ryan

it all worked when I made the masters not schedulable. I had issues when I made the masters schedulable, the console and authentication didn't come up afterward. In the written guide it was not clear that it is **mandatory** to run:

`sed -i 's/mastersSchedulable: true/mastersSchedulable: false/' ~/ocp-install/manifests/cluster-scheduler-02-config.yml`

When reading it first, it sounds like it is optional to run the command. I just think it should be made a bit more clear 👍 .
